.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+# Motivation
+
+<!-- Describe the motivation that lead to the PR -->
+
+# Changes
+
+<!-- List the changes that have been developed -->
+
+# Tests
```

Comment:
I look forward to the time when I can look at this section and actually be confident that there are no bugs!